tools/autograd/gen_inplace_or_view_type.py

```diff
@@ -346,24 +344,13 @@ def get_view_info(f: NativeFunction) -> Optional[str]:
     return view_info
 
 
-# For view replay calls, we generate an ordinary Dispatcher::call() instead, because:
-#  - We want to replay the entire call into the op, including any previously-set dispatch keys (including autograd!).
-#  - The view replay call also is not part of the hot path.
-def emit_view_call(
-    f: NativeFunction, input_base: str, unpacked_args: Sequence[str]
-) -> str:
-    # View replay functions use the standard Dispatcher::call API.
-    return CALL_DISPATCH.substitute(
-        unambiguous_name=f.func.name.unambiguous_name(), unpacked_args=unpacked_args
-    )
-
-
-def emit_view_lambda(
+def emit_view_func(
     f: NativeFunction, bindings: List[Binding], view_idx: Optional[str] = None
 ) -> str:
     """Generate an additional lambda function to recover views in backward when as_strided is not supported.
     See Note [View + Inplace update for base tensor] and [View + Inplace update for view tensor] for more details.
     """
+    # TODO: Clean this logic up if we get rid of reverse view funcs or reify them.
```

Comment:
Can we get rid of reverse view funcs now?

Reply:
Sure I'm happy to do it, preference on whether it's done in this PR or a separate one?

Reply:
I think ideally it would have been a revert PR before this one, but I don't know if that minimizes the work for you. I would opt for work minimization